cmd/acr-credential-provider/main.go

```diff
@@ -60,6 +63,9 @@ func main() {
 	logs.InitLogs()
 	defer logs.FlushLogs()
 
+	// Flags
+	command.Flags().StringVarP(&mirrorMappingStr, "mirror-mapping", "m", "", "mirror mapping to use")
```

Comment:
this flag value is defined after it's used in L50, then how could external user use defined flag to override this?

Reply:
I think as long as it is before `command.Execute()` then it is ok.

Please check the CCM code. You can see cmd is defined first, then flags.
https://github.com/kubernetes-sigs/cloud-provider-azure/blob/9eb8173ac1c320f227b51c333e37d3b7e60c899e/cmd/cloud-controller-manager/app/controllermanager.go#L79-L159